Source/Core/VideoBackends/OGL/OGLTexture.cpp

```diff
@@ -120,17 +117,13 @@ OGLTexture::OGLTexture(const TextureConfig& tex_config) : AbstractTexture(tex_co
     // method is in the base renderer class and can be called by VideoCommon.
     FramebufferManager::SetFramebuffer(0);
   }
-
-  SetStage();
 }
 
 OGLTexture::~OGLTexture()
 {
   if (m_texId)
   {
-    for (auto& gtex : s_Textures)
-      if (gtex == m_texId)
-        gtex = 0;
+    g_renderer->UnbindTexture(this);
```

Comment:
Isn't this handled by the base AbstractTexture class?

Reply:
Yep, good catch.